app/helpers/admin/dry_helper.rb

```diff
@@ -1,19 +1,26 @@
 module Admin
   module DryHelper
     BOOL_BG_COLOR_CLASS = {
-      true  => 'bg-success',
-      false => 'bg-danger'
+      true  => 'green',
```

Comment:
do you like [green gray band](http://modna.com.ua/images/fact/green-grey-poedut-na-mtv-ema.jpg)  ?